src/main/java/io/voucherify/client/model/order/OrderItemProduct.java

```diff
@@ -0,0 +1,26 @@
+package io.voucherify.client.model.order;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Singular;
+import lombok.ToString;
+
+import java.util.Map;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@Getter
+@Builder
+@ToString
+public class OrderItemProduct {
+
+    private String name;
+
+    @Singular("metadataEntry")
+    private Map<String, Object> metadata;
+
+    private boolean override = false;
```

Comment:
@lkulig We can't change this property to `true` as far as I see, can we?

Reply:
@tpindel we can, by default it's set to false but `Builder`  for this class will expose the `override` method that will allow someone to change it as they wish

Reply:
👍